core/templates/services/exploration-features-backend-api.service.ts

```diff
@@ -32,7 +32,7 @@ interface ExplorationFeaturesBackendDict {
 
 export interface ExplorationFeatures {
   isExplorationWhitelisted: boolean;
-  alwaysAskLearnersForAnswerDetails: boolean;
+  alwaysAskLearnersForAnswerDetails?: boolean;
```

Comment:
Why this needs to be optional?

Reply:
![image](https://user-images.githubusercontent.com/46513107/113520312-c97c6c00-9557-11eb-979c-7093d873e51d.png)
This interface is used in the exploration-feature.service.spec.ts, and the *alwaysAskLearnersForAnswerDetails* property is not there.

Reply:
Hmm, okay, if it is just in tests then make sure to fill this out in the test files and not mark it as optional. This applies to any other places.

Reply:
@vojtechjelinek Maybe like this?

![image](https://user-images.githubusercontent.com/46513107/113730290-60166d80-96bd-11eb-9d4c-3b4d129c1c73.png)

Adding comments to explain why that property is there